docs/releases/1.10.txt

```diff
@@ -53,6 +53,8 @@ Minor features
   to prevent an issue where Safari caches redirects and prevents a user from
   being able to log out.
 
+* The :func:`~django.contrib.auth.views.login` view has a new parameter ``redirect_authenticated_user`` to automatically redirect users opening the view as if they had just logged in.
```

Comment:
This line needs to be wrapped at 80 chars; see the [guidelines for writing Django docs](https://docs.djangoproject.com/en/1.8/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files).